scipy/interpolate/interpolate.py

```diff
@@ -22,12 +22,13 @@
 from ._bsplines import make_interp_spline, BSpline
 
 
-def lagrange(x, w):
+def lagrange(x, y, reduce=True):
     r"""
-    Return a Lagrange interpolating polynomial.
+    poly = lagrange(x, y)
```

Comment:
This line is breaking the docstring rendering: the first line is a full sentence of a short description of a function. Best roll back this change.